src/components/todolistcrud.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] **Optional:** Rather than disable the linter strive to fix them as this makes you more accustomed to the style guide of the tech.